torch/_functorch/aot_autograd.py

```diff
@@ -596,19 +597,29 @@ class ViewAndMutationMeta:
     # TODO: we should kill this
     # (need to default it to not break internal)
     is_train: bool = False
+    # Whether we are tracing through subclasses
+    # (ideally should remove this)
+    is_subclass: bool = False
```

Comment:
I think this path is triggered only when there is some or all inputs are subclasses. In that case, we do the default thing where we just return the mutated inputs instead of trying to keep input mutations in the graph.

Reply:
bump on this rename :)